vehicle-interface/firmware.mkd

```diff
@@ -38,15 +35,14 @@ it.
 
 To read expanded data from a particular car, look for firmware distributed as
 pre-compiled binaries by your car's manufacturer. Check the [list of supported
-vehicles](/hardware/vehicles.html) for your car's make and model and for links
+vehicles](http://openxcplatform.com/hardware/vehicles.html) for your car's make and model and for links
```

Comment:
The relative link gave me a 404 error of Github. Should I make it a separate commit to the file or leave it out?